src/main/java/com/github/vkorobkov/jfixturescmd/SqlTypes.java

```diff
@@ -0,0 +1,25 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import java.util.EnumSet;
+import java.util.stream.Collectors;
+
+public enum SqlTypes {
+    POSTGRES,
+    MYSQL,
+    H2,
+    CLICKHOUSE;
+
+    public static String getValues() {
+        return EnumSet.allOf(SqlTypes.class)
+                .stream()
+                .map(Enum::name)
+                .collect(Collectors.joining(", "));
+    }
+
+    public static SqlTypes fromString(final String value) {
```

Comment:
Let's create a utility class called `EnumUtil` and put a method in there:
```
<E extends Enum<E>> Optional<E> valueOf(Class<E> enumClazz, String value) {
        return EnumSet.allOf(enumClazz).stream().filter(type -> type.name().equals(value)).findFirst();
    }
```
The caller, `SqlTypesConverter` would upper case the value before call EnumUtil